sdl2_mixer/VITABUILD

```diff
@@ -1,37 +1,43 @@
 pkgname=sdl2_mixer
 pkgver=2.6.3
-pkgrel=3
+pkgrel=4
```

Comment:
this is no more 2.6.3 (because sdl2 has two different branches `SDL2` & `release-2.6.x`
- if you want to **keep** the 2.6.x version numbering, you have to make the patches for cherry picking or have to fork the repo for self maintaining.
- if you want to always follow the nightly dev version, you have to make the different VITABUILD (`sdl2_mixer-git` and have to contain `conflict` info with `sdl2_mixer`) or have to change the pkgver & pkgrel (`9999`)

Reply:
Neither of those. I changed to 2.7.0.

Reply:
it's `2.7.0-pre` not `2.7.0`
you can't use `2.7.0-1`
well.. it could be just a logical thing. i mean, this version is uncleared/invalid information.

Reply:
Yes i can. We had this scheme with sdl2 for months.

Reply:
you already updated pr, so this thread is totally outdated...
btw https://github.com/vitasdk/packages/pull/225 i thought it was patched to released tree
but current libsdl has multiple trees; next upstream and current release tree.

anyway, thank you for applying my suggestion.